engines/scumm/imuse_digi/dimuse_internalmixer.cpp

```diff
@@ -86,7 +86,7 @@ int IMuseDigiInternalMixer::init(int bytesPerSample, int numChannels, uint8 *mix
 	// at different levels of granularity depending on the case (as uint32, uint16 or uint8).
 	// The latter is populated from the middle, inside-out; hence _softLMID.
 
-	int *tableMalloc = (int *)malloc(53376 * sizeof(int));
+	int *tableMalloc = (int *)malloc(53376 * sizeof(int32));
```

Comment:
int->int32 :)

Reply:
...I was very confused, and then I looked at the file and I noticed I didn't save it before the latter commit :)